### PR TITLE
2046 update nuget for token validation

### PR DESCRIPTION
**Pull request by sahma19**

### CODE REVIEW

File: libraries/dotnet/EnergyOrigin.TokenValidation/EnergyOrigin.TokenValidation.Unit.Tests/Options/TokenValidationOptionsTests.cs [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+            Issuer = issuer,
+            Audience = audience
+        };
+
+        var validationResults = new List<ValidationResult>();
+        var validationContext = new ValidationContext(options, null, null);
+        var isValid = Validator.TryValidateObject(options, validationContext, validationResults, true);
+
+        Assert.False(isValid);
+        Assert.NotEmpty(validationResults);
```
Comment (CodeReaper) [inline libraries/dotnet/EnergyOrigin.TokenValidation/EnergyOrigin.TokenValidation.Unit.Tests/Options/TokenValidationOptionsTests.cs:55]: You should check equals with the validation success property instead

Comment (sahma19) [reply libraries/dotnet/EnergyOrigin.TokenValidation/EnergyOrigin.TokenValidation.Unit.Tests/Options/TokenValidationOptionsTests.cs]: Like this?: https://github.com/Energinet-DataHub/energy-origin/pull/513/commits/214872e553cafd4e129295bb87a811f08d0f82da

Comment (CodeReaper) [reply libraries/dotnet/EnergyOrigin.TokenValidation/EnergyOrigin.TokenValidation.Unit.Tests/Options/TokenValidationOptionsTests.cs]: Well no, I mean check equals against this property: https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.validationresult.success?view=net-8.0

Comment (sahma19) [reply libraries/dotnet/EnergyOrigin.TokenValidation/EnergyOrigin.TokenValidation.Unit.Tests/Options/TokenValidationOptionsTests.cs]: Like this then?:

https://github.com/Energinet-DataHub/energy-origin/pull/513/commits/90a3893630db1ed99d4489fa2542cff54659517f

**Review by CodeReaper — APPROVED** [review]

**Pull request merged**